Extractor/FrontendExtractor.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle\Extractor;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Translation\Extractor\AbstractFileExtractor;
+use Symfony\Component\Translation\MessageCatalogue;
+use Symfony\Component\Translation\Extractor\ExtractorInterface;
+
+final class FrontendExtractor extends AbstractFileExtractor implements ExtractorInterface
+{
+    const FUNCTION_STRING_ARGUMENT_REGEX = '\s?[\'"]([^"\'),]+)[\'"]\s?';
+    const REGEX_DELIMITER = '/';
+
+    private $prefix = '';
+
+    private $filesystem;
+
+    private $configuration;
+
+    public function __construct(Filesystem $filesystem, array $configuration = []) {
+        $this->filesystem = $filesystem;
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function extract($resource, MessageCatalogue $catalogue)
+    {
+        $files = $this->extractFiles($resource);
+
+        foreach ($files as $file) {
+            foreach ($this->configuration as $configuration)
+            {
+              $pathInfo = pathinfo($file);
+              if (in_array($pathInfo['extension'], $configuration['extensions'], true))
+              {
+                $this->parseMessagesFromContent(file_get_contents($file), $catalogue, $configuration['sequence']);
```

Comment:
there is nothing ensuring that `$configuration['sequence']` will exist here (if you omit it from the bundle config file when defining the extractor, you won't get any configuration error, and you will get a notice here)

Reply:
OK